cpp/src/ray/util/process_helper.cc

```diff
@@ -57,6 +60,18 @@ void ProcessHelper::StartRayNode(const int redis_port, const std::string redis_p
     cmdargs.emplace_back("--resources");
     cmdargs.emplace_back(FormatResourcesArg(resources));
   }
+  if (include_dashboard) {
+    cmdargs.emplace_back("--include-dashboard");
```

Comment:
the cmdargs init with {"--include-dashboard", "false"} before, here emplace it again?

Reply:
I lost to remove the old command, thanks.